[Ready to Merge] Vs docker tools config

ghogen opened the pull request:
## Summary

Incorporate product team feedback on the VS Docker Tools container eBook topic.

ghogen committed:
add docker tools options info and image

ghogen committed:
Add info about applicability to config table

ghogen committed:
merge master

ghogen commented:
This is a work in progress for now, don't merge yet.

ghogen committed:
remove extra text

ghogen committed:
Add warning

ghogen committed:
tweak wording

rpetrusha commented on docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md:
```diff
@@ -70,12 +70,15 @@ Figure 4-30: Docker Tools Options
+| Do not prompt for trusting localhost SSL certificate | Off | ASP.NET Core 2.1 projects | If the localhost SSL certificate is not trusted, Visual Studio will prompt every time you run your project, unless this checkbox is checked. |
+
+> [!WARNING]
+> If the localhost SSL certificate is not trusted, and you check the box to suppress prompting, then your project can't run. In that case, uncheck the **Do not prompt** checkbox, run your project, indicate trust at the prompt.
```
an "and" should separate the comma and "indicate"

ghogen replied:
Thanks

rpetrusha reviewed (changes requested):
This looks good, @ghogen. I've just left one minor comment.

ghogen committed:
address Ron's feedback

ghogen commented:
Waiting on PM review.

ghogen committed:
update warning message

ghogen commented:
I incorporated review, so this is good now.  @mairaw I don't have permissions to change labels, but I incorporated review feedback, so this is ready to merge.

ghogen commented:
@rpetrusha This is good to merge now.
